drivers/mssqql.go

```diff
@@ -275,22 +277,54 @@ func (db *MSSQL) GetRecords(database, table, where, sort string, offset, limit i
 		if err := rows.Scan(rowValues...); err != nil {
 			return nil, 0, err
 		}
-
+		// Get column types to identify UNIQUEIDENTIFIER
+		columnTypes, err := rows.ColumnTypes()
+		if err != nil {
+			return nil, 0, err
+		}
```

Comment:
I'm paranoid, but to avoid panic when accessing columnTypes[i] a few lines below, I would add this


```suggestion
		columnTypes, err := rows.ColumnTypes()
		if err != nil {
			return nil, 0, err
		}

		if len(columnTypes) != len(rowValues) {
			return nil, 0, errors.New("unexported number of column")
		}
```

I dislike panic

This was what I reported here

https://github.com/jorgerojas26/lazysql/pull/174#discussion_r1995484157

But my suggestion wasn't working 

```go
columnType, ok := columnTypes[i]
if !ok {
}
```

This one works for maps, not slices